int32 T
int32 nXr
int32 nXh

bool tracking_lost

float64[] xr_init
float64[] xh_init
float64[] robot_traj_opt
float64[] human_traj_hp_opt
float64[] human_traj_rp_opt